Account/TokenServerClient.swift

```diff
@@ -175,14 +174,14 @@ open class TokenServerClient {
                 return
             }
 
-            let json = JSON(data)
+            let jsonDict = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any]
```

Comment:
nice work, could we add some error handling here to catch errors if try fails

@imsanchit 

cc @lmarceau

Reply:
Thanks for the suggestion @nbhasin2, have updated the PR and added error handling. Please have a look